app/helpers/md-regex-replace.js

```diff
@@ -0,0 +1 @@
+export { default, mdRegexReplace } from 'ember-cli-materialize/helpers/md-regex-replace';
```

Comment:
I'd just use an obviously private namespace like `{{mdPRIVATE}}`. This is part of what I need to think about RE: the select component. If your approach didn't rely on helpers for so much functionality (and used CPs for example, we could keep everything contained within the addon a little better).